index.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+require('vendor/autoload.php');
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+use Nexmo\Client;
+use Nexmo\Client\Credentials\Keypair;
+use Aws\S3\S3Client;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+use Aws\TranscribeService\TranscribeServiceClient;
+
+Dotenv\Dotenv::create(__DIR__)->load();
+
+$app = AppFactory::create();
+
+/**
+ * Answer the call and prompt using NCCO flow
+ */
+$app->any('/webhooks/answer', function (Request $request, Response $response) {
+    $uri = $request->getUri();
+
+    if ($request->getMethod() != 'GET') {
+        return $response->withStatus(403);
+    }
+
+    // NOTE: The usage of $uri below will not work when using ngrok, as getHost() will return localhost
```

Comment:
This is true if you access your application on localhost. If you visit it using the `ngrok` URL it will be populated as intended as it respects the `Host` header

Reply:
Unfortunately, in development ngrok was not returning the correct URL. It was returning "localhost" instead of the ngrok URL. Therefore, I found it necessary to hardcode the address.

Reply:
Odd, this should work. I just tested through ngrok and got `http://406f4270.ngrok.io` back as my host, which was my ngrok tunnel at the time. Were you running this under Docker?

Reply:
I think I was originally testing that method using localhost, then just kept going with the assumption when starting to use ngrok.

Reply:
Removing comment in a new commit.